.gitlab-ci.yml

```diff
@@ -101,7 +101,7 @@ run-tests:
   script:
     # Try to pull from dockerhub, if it fails, the run should fail
     - tools/docker/image-pull.sh
-    - tests/test_flows.py -v -u $RANDOM
+    - tests/test_flows.py -v -u $RANDOM --tcpdump
```

Comment:
Can we make tcpdump enabled by default? Any reason not to?

Reply:
Well, it will be once #1020 matures.